pgbouncer/client.go

```diff
@@ -29,3 +30,9 @@ type Client struct {
 type XormClient struct {
 	*xorm.Engine
 }
+
+type XormClientList struct {
+	List  []*XormClient
```

Comment:
Add some uninported fields here : 
```
pb *api.PgBouncer, ctx context.Context, podName string, auth *Auth, dbType string, dbName string
```
And use it while calling `addXormClient()` instead of a bunch of params

Reply:
done!